bindgen/callbacks.rs

```diff
@@ -105,7 +105,15 @@ pub trait ParseCallbacks: fmt::Debug {
     ///
     /// If no additional attributes are wanted, this function should return an
     /// empty `Vec`.
-    fn add_derives(&self, _name: &str) -> Vec<String> {
+    fn add_derives(&self, _info: DeriveInfo<'_>) -> Vec<String> {
```

Comment:
Let's pass a `&DeriveInfo` instead? If we need some expensive-to-copy data there we could avoid the copy trivially.